### PR TITLE
Feature/documentation update

### PR DESCRIPTION
**Pull request by corradot12**
[XXXX-<Title> - Please use the Work Item number and Title as PR Name, not subtasks]

#### 📲 What

A description of the change.

#### 🤔 Why
		
Why it's needed, background context.
		
#### 🛠 How
		
More in-depth discussion of the change or implementation.

#### 👀 Evidence
		
Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR
		 
#### 🕵️ How to test

Notes for QA

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: src/tests/Performance/xxAMIDOxx.xxSTACKSxx.API.PerformanceTests/ReadMe.md [hunk 1]
```diff
@@ -1,59 +1,3 @@
...
-
-This folder structure and the contents of each folder follows the guideline outlined in the blog post on 
-[Testing Excellence](https://www.testingexcellence.com/gatling-maven-performance-test-framework/).
-It is recommended that users of this framework refer back to this blog post, as it contains all required details.
-
-_NB: Testing Excellence was written using Gatling 2.3.0, whereas this project is using the current latest version 3.0.3. 
-Contrary to what the blog post says, to run the tests you should use `mvn clean gatling:test` rather than `mvn clean gatling:execute`.
-More details here: (https://gatling.io/docs/current/migration_guides/2.3-to-3.0)_
\ No newline at end of file
+The full documentation on Amido Stacks can be found [here](https://amido.github.io/stacks/).
```
Comment (VitalinaVZdrobau) [inline src/tests/Performance/xxAMIDOxx.xxSTACKSxx.API.PerformanceTests/ReadMe.md:3]: maybe it's worth providing the more specific link to the document location?
in this case:
"https://amido.github.io/stacks/docs/workloads/azure/backend/netcore/testing/performance_testing_netcore/"

**Review by VitalinaVZdrobau — APPROVED** [review]

File: src/tests/Performance/xxAMIDOxx.xxSTACKSxx.API.PerformanceTests/ReadMe.md (hunk 1 above)
Comment (corradot12) [reply src/tests/Performance/xxAMIDOxx.xxSTACKSxx.API.PerformanceTests/ReadMe.md:3]: Should the performance testing be in this repository in the first place?
I added the specific link in some readmes

Comment (VitalinaVZdrobau) [reply src/tests/Performance/xxAMIDOxx.xxSTACKSxx.API.PerformanceTests/ReadMe.md:3]: Most probably - no, but let's confirm that with Jack Sheppard.
Also, as the current solution does not generate the UI tests anymore, the `/stacks-dotnet/src/ui` folder should be deleted too.

Comment (jpbsheppard) [reply src/tests/Performance/xxAMIDOxx.xxSTACKSxx.API.PerformanceTests/ReadMe.md:3]: Performance was originally included as the .net solution was the only repository. It has now been pulled out as a standalone solution.  I would keep this in for now though

**Pull request merged**